src/course-updates/CourseUpdates.jsx

```diff
@@ -37,88 +44,115 @@ const CourseUpdates = ({ courseId }) => {
+                        {courseUpdates.length ? courseUpdates.map((courseUpdate, index) => (
+                          isInnerFormOpen(courseUpdate.id) ? (
+                            <UpdateForm
+                              isOpen={isUpdateFormOpen}
+                              close={closeUpdateForm}
+                              requestType={requestType}
+                              isInnerForm
+                              isFirstUpdate={index === 0}
+                              onSubmit={handleUpdatesSubmit}
+                              courseUpdatesInitialValues={courseUpdatesInitialValues}
+                            />
+                          ) : (
+                            <CourseUpdate
+                              dateForUpdate={courseUpdate.date}
+                              contentForUpdate={courseUpdate.content}
+                              onEdit={() => handleOpenUpdateForm(REQUEST_TYPES.edit_update, courseUpdate)}
+                              onDelete={() => handleOpenDeleteForm(courseUpdate)}
+                              isDisabledButtons={isUpdateFormOpen}
+                            />
+                          ))) : null}
+                      </div>
+                      <div className="updates-handouts-container">
+                        <CourseHandouts
+                          contentForHandouts={courseHandouts?.data || ''}
+                          onEdit={() => handleOpenUpdateForm(REQUEST_TYPES.edit_handouts)}
+                          isDisabledButtons={isUpdateFormOpen}
+                        />
+                      </div>
+                      <DeleteModal
+                        isOpen={isDeleteModalOpen}
+                        close={closeDeleteModal}
+                        onDeleteSubmit={handleDeleteUpdateSubmit}
                       />
+                      {isShowProcessingNotification && (
+                        <ProcessingNotification
+                          isShow={isShowProcessingNotification}
+                          title={processingNotificationTitle}
+                        />
+                      )}
                     </div>
-                    <DeleteModal
-                      isOpen={isDeleteModalOpen}
-                      close={closeDeleteModal}
-                      onDeleteSubmit={handleDeleteUpdateSubmit}
-                    />
-                  </div>
-                </section>
-              </div>
-            </article>
-          </Layout.Element>
-        </Layout>
-      </section>
-    </Container>
+                  </section>
+                </div>
+              </article>
+            </Layout.Element>
+          </Layout>
+        </section>
+      </Container>
+      <div className="alert-toast">
+        <InternetConnectionAlert
+          isFailed={anyStatusFailed}
+          isQueryPending={anyStatusInProgress || anyStatusPending}
+          onInternetConnectionFailed={() => null}
```

Comment:
Do we need to change the state if there was an error?

Reply:
I think no, you mean some specific state or in general?

Reply:
`savingStatus` or `loadingStatus` to `RequestStatus.FAILED` to show error or something else?